src/Modal/Modal.component.tsx

```diff
@@ -96,11 +80,32 @@ const castStyles = {
   },
 };
 
+const modalSizeRules: Function = (
+  modalSize:string,
+  theme: any,
+) => {
+  switch (modalSize) {
+    case ('full'):
```

Comment:
When modalSize if fullscreen, the footer seems to occupy upto 90% of the modal height. Design doc shows that the modal body should occupy the larger space.

Reply:
@Dantiff For this, the body can occupy as much space as it needs while pushing the footer down